src/TestProblem/Hydro/Riemann/Init_TestProb_Hydro_Riemann.cpp

```diff
@@ -487,6 +492,50 @@ void SetBFieldIC( real magnetic[], const double x, const double y, const double
 
 } // FUNCTION : SetBFieldIC
 #endif // #ifdef MHD
+
+
+
+#if ( EOS == EOS_NUCLEAR  &&  NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP )
+//-------------------------------------------------------------------------------------------------------
+// Function    :  Flu_ResetByUser_Riemann
+// Description :  Function to reset the temperature initial guess
+//
+// Note        :  1. Invoked by "Flu_ResetByUser_API()" and "Model_Init_ByFunction_AssignData()" using the
+//                   function pointer "Flu_ResetByUser_Func_Ptr"
+//                2. This function will be invoked when constructing the initial condition
+//                    (by calling "Model_Init_ByFunction_AssignData()") and after each update
+//                    (by calling "Flu_ResetByUser_API()")
+//                3. Input "fluid" array stores the original values
+//                4. Even when DUAL_ENERGY is adopted, one does NOT need to set the dual-energy variable here
+//                   --> It will be set automatically in "Flu_ResetByUser_API()" and "Model_Init_ByFunction_AssignData()"
+//                5. Enabled by the runtime option "OPT__RESET_FLUID"
+//
+// Parameter   :  fluid    : Fluid array storing both the input (origial) and reset values
+//                           --> Including both active and passive variables
+//                Emag     : Magnetic energy (MHD only)
+//                x/y/z    : Target physical coordinates
+//                Time     : Target physical time
+//                dt       : Time interval to advance solution
+//                lv       : Target refinement level
+//                AuxArray : Auxiliary array
+//
+// Return      :  true  : This cell has been reset
+//                false : This cell has not been reset
+//-------------------------------------------------------------------------------------------------------
+bool Flu_ResetByUser_Riemann( real fluid[], const double Emag, const double x, const double y, const double z, const double Time,
+                              const double dt, const int lv, double AuxArray[] )
+{
+
+      const bool   CheckMinTemp_No = false;
```

Comment:
Remove the extra spaces between `bool` and `CheckMinTemp_No`.

Reply:
I removed them.